k-distribution/tests/regression-new/checks/badFunctionRuleWithContext.k.out

```diff
@@ -1,6 +1,6 @@
 [Error] Compiler: Had 1 parsing errors.
-[Error] Critical: Unexpected sort Bool for term parsed as production syntax
-Bool ::= "false" [token()]. Expected Int.
+[Error] Inner Parser: Unexpected sort Bool for term parsed as production syntax
+Bool ::= "false" [token()]. Expected: Int
```

Comment:
Can we make these error messages easier to read while we're at it?

I would like to see:

```
[Error] Inner Parser: Unexpected sort for term:
    production: syntax Bool ::= "false" [token()].
    expected: Int
    found: Bool
    Source(badFunctionRuleWithContext.k)
    Location(10,23,10,28)
```

Reply:
This is a larger change than it sounds and would involve changing the general mechanism for how we format errors to the user. I'd rather we make it separately if we're going to make a change like this, as this PR is already huge.